src/EditorFeatures/CSharpTest2/Recommendations/RecordKeywordRecommenderTests.cs

```diff
@@ -3,13 +3,14 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
 using Microsoft.CodeAnalysis.Test.Utilities;
 using Roslyn.Test.Utilities;
 using Xunit;
 
 namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Recommendations
 {
-    public class DataKeywordRecommenderTests : KeywordRecommenderTests
+    public class RecordKeywordRecommenderTests : KeywordRecommenderTests
```

Comment:
>RecordKeywordRecommenderTests [](start = 17, length = 29)

:memo: this file is a duplicate of ClassKeywordRecommenderTests with one or two tests added, such as typing in position nested in record.